source/includes/documentation/_vue_js_elements.md

```diff
@@ -2038,47 +2038,98 @@ For examples of usage see: [Navigation Drawers tests](https://github.com/jdi-tes
-      cartIconWithTooltip.click();
-      tooltip.is().hidden();
+      tooltip.has().color(color);
     }
 ```
 
-Tooltip component is useful for conveying information when a user hovers over an element. Its display can be controlled programmatically. When activated, tooltips display a text label identifying an element, such as a description of its function.
+__Tooltip__ - Tooltip is useful for conveying information when a user hovers over an element.
+Its display can be controlled programmatically.
+When activated, tooltips display a text label identifying an element, such as a description of its function.
 
 ![Tooltips examples](../../images/vuetify/tooltips.png)
 
-|Method | Description | Return Type
---- | --- | ---
-**is()** | Returns Assert class | TooltipAssert
-
 For examples of usage see: [Tooltips tests](https://github.com/jdi-testing/jdi-light/blob/vuetify-develop/jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/TooltipsTests.java).
 
+__Vuetify v2.6.14__ code example:
+
+```html
+<template>
+  <div class="text-center d-flex align-center justify-space-around">
+    <v-tooltip bottom>
+      <template v-slot:activator="{ on, attrs }">
+        <v-btn
+          color="primary"
+          dark
+          v-bind="attrs"
+          v-on="on"
+        >
+          Button
+        </v-btn>
+      </template>
+      <span>Tooltip</span>
+    </v-tooltip>
+
+    <v-tooltip bottom>
+      <template v-slot:activator="{ on, attrs }">
+        <v-icon
+          color="primary"
+          dark
+          v-bind="attrs"
+          v-on="on"
+        >
+          mdi-home
+        </v-icon>
+      </template>
+      <span>Tooltip</span>
+    </v-tooltip>
+
+    <v-tooltip bottom>
+      <template v-slot:activator="{ on, attrs }">
+        <span
+          v-bind="attrs"
+          v-on="on"
+        >This text has a tooltip</span>
+      </template>
+      <span>Tooltip</span>
+    </v-tooltip>
+  </div>
+</template>
+```
+
+```html
+<div data-app="true" id="app" class="v-application v-application--is-ltr theme--light">
+  <div class="v-application--wrap">...</div>
+  <div class="v-tooltip__content" style="left: 436px; top: 242px; z-index: 8; display: none;"> <span>Tooltip for "Button"</span></div>
+  <div class="v-tooltip__content" style="left: 313px; top: 309px; z-index: 8; display: none;"> <span>Left tooltip</span></div>
+</div>
+```
+
+| Method         | Description                                            | Return Type   |
+|----------------|--------------------------------------------------------|---------------|
+| **is()/has()** | Returns Assert class                                   | TooltipAssert |
+| **color()**    | Returns css attribute background-color as String Value | String        |
+
+<a href="https://github.com/jdi-testing/jdi-light/blob/vuetify-develop/jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/TooltipsTests.javas">Java test examples</a>
+
```

Comment:
Hi, the link does not work :(

Reply:
Thank you for this comment! It was a misprint, fixed